LeaderboardBackend.Test/Features/Users/LoginTests.cs

```diff
@@ -0,0 +1,130 @@
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using LeaderboardBackend.Authorization;
+using LeaderboardBackend.Models.Entities;
+using LeaderboardBackend.Models.Requests;
+using LeaderboardBackend.Test.Fixtures;
+using LeaderboardBackend.Test.Lib;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using NUnit.Framework;
+
+namespace LeaderboardBackend.Test.Features.Users;
+
+public class LoginTests : IntegrationTestsBase
+{
+    private const string VALID_EMAIL = "valid@user.com";
+    private const string BANNED_EMAIL = "banned@user.com";
+    private const string VALID_PASSWORD = "P4ssword";
```

Comment:
Same as elsewhere, reserve the use of `const` for unchanging constants. Use `readonly` instead. There is no performance benefit to marking variables as constant in C#.

Reply:
Performance wasn't my concern here, but the fact that the `TestCase` attributes don't like it when their arguments aren't `const`s. Swapping them to `readonly` makes even declaring these fields a bit useless, IMO, as they get very little use outside of the attribute args.

Reply:
That "best practice" is only for public constants.
This is fine.